File: express/app/src/main/java/com/changsukuaidi/www/modules/login/LoginActivity.java
package com.changsukuaidi.www.modules.login;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.changsukuaidi.www.base.BaseActivity;

/**
 * @Describe
 * @Author thc
 * @Date 2017/12/6
 */

public class LoginActivity extends BaseActivity<LoginFragment> {
    @Override
    protected LoginFragment getFragment() {
        return new LoginFragment();
    }

    @Override
    protected void initData() {

    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        BaseActivity.clearTaskStayOne(this);
    }
}

File: WIFIDirect/app/src/main/java/com/thc/www/p2p/android/login/LoginActivity.kt
package com.thc.www.p2p.android.login

import com.thc.www.p2p.base.BaseActivity

/**
 * @Describe
 * @Author thc
 * @Date  2018/5/7
 */
class LoginActivity : BaseActivity<LoginFragment>() {
    override fun getFragment(): LoginFragment = LoginFragment()
}